umm-spec-lib/src/cmr/umm_spec/test/expected_conversion_util.clj

```diff
@@ -13,6 +14,23 @@
   [cmr.umm-spec.url :as url]
   [cmr.umm-spec.util :as su]))
 
+(def coll-progress-enum-list
+  "The enum list for CollectionProgress in v1.10. that could be converted from 
+   all formats except for DIF10"
+  (set ["PLANNED" "ACTIVE" "COMPLETE" "NOT PROVIDED" "NOT APPLICABLE"]))
+
+(defn expected-coll-progress
```

Comment:
It is needed. Two cases:
1. for v1.10 case.  UMM contains all the valid enums, say it translates to dif10, which only supports 3 out of 5 enums. 
   so when you do umm-dif10 and back to umm,  the non-supported enums become "NOT PROVIDED" - according to the mappings in the ticket.